### PR TITLE
add target folder control to the generator prompt 

### PR DESCRIPTION
**Pull request by slavik-lvovsky**

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -32,12 +32,21 @@
...
+
           <PromptInfo v-if="currentPrompt && !isDone" :currentPrompt="currentPrompt" />
+          <div style="width:50%;">
+            <Form
+              ref="folderForm"
+              :questions="folderBrowserQuestions"
+              v-show="shouldShowGeneratorSelection()"
+              @answered="setTargetFolder"
+            />
+          </div>
```
Comment (tomer-epstein) [inline frontend/src/App.vue:46]: TargetFolder.vue

File: frontend/src/App.vue [hunk 2]
```diff
@@ -32,12 +32,21 @@
...
+
           <GeneratorSelection
             v-if="shouldShowGeneratorSelection()"
             @generatorSelected="selectGenerator"
-            :currentQuestion="currentPrompt.questions[0]"
+            :currentQuestion="currentPrompt.questions[1]"
           />
           <v-slide-x-transition>
```
Comment (tomer-epstein) [inline frontend/src/App.vue:53]: Questions.vue

**Comment by tomer-epstein** [comment]
[UI] x 3

image

**Comment by slavik-lvovsky** [comment]
All relevant strings will be changed by me and Paola once the code is ready

File: frontend/src/App.vue [hunk 3]
```diff
@@ -70,7 +54,7 @@
           <div class="diagonal">
           </div>
           <div class="bottom-buttons-col" style="display:flex;align-items: center;">
-            <v-btn id="back" :disabled="promptIndex<1 || isReplaying" @click="back" v-show="false && !shouldShowGeneratorSelection()">
+            <v-btn id="back" :disabled="promptIndex<1 || isReplaying" @click="back" v-show="false">
```
Comment (tomer-epstein) [inline frontend/src/App.vue:57]: there should be a flag for not showing 'back' btn at 1st page.

Comment (slavik-lvovsky) [reply frontend/src/App.vue]: Done

**Pull request merged**